src/main/java/com/pingcap/tikv/codec/CodecDataInput.java

```diff
@@ -179,6 +207,17 @@ public String readUTF() {
     }
   }
 
+  public int peekByte() {
+    mark(currentPos());
```

Comment:
I believe we don't need to get `currentPos` because `mark()` for `ByteArrayInputStream` uses its own position for marking.

Reply:
We might replace totally byteArrayInputStream later. Existing function signature doesn't use the mark is quite wired behavior and I'd prefer to remain passing correct mark in case future replacement.